test/index.ts

```diff
@@ -176,7 +176,87 @@ describe('Storage', () => {
         autoRetry,
       });
       const calledWith = storage.calledWith_[0];
-      assert.strictEqual(calledWith.autoRetry, autoRetry);
+      assert.strictEqual(calledWith.retryOptions.autoRetry, autoRetry);
+    });
+
+    it('should propagate autoRetry in retryOptions', () => {
+      const autoRetry = false;
+      const storage = new Storage({
+        projectId: PROJECT_ID,
+        retryOptions: {autoRetry},
+      });
+      const calledWith = storage.calledWith_[0];
+      assert.strictEqual(calledWith.retryOptions.autoRetry, autoRetry);
+    });
+
+    it('should throw if autoRetry is defined twice', () => {
+      const autoRetry = 10;
+      assert.throws(() => {
+        const storage = new Storage({
```

Comment:
Since this is testing that the constructor throws an exception could the assignment to the const be removed to cleanup the linter warning? Same goes for L285.

Reply:
whoops, totally missed the linter warnings. fixed